src/apps/frontend/components/dropdowns/dropdown-default.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+
+import TaskPopup from '../task/task-popup';
+
+const DropdownDefault: React.FC = () => {
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const [popupOpen, setPopupOpen] = useState(false);
+
+  return (
+    <div className="relative flex">
+      <button
+        className="text-[#98A6AD] hover:text-body"
+        onClick={() => setDropdownOpen(!dropdownOpen)}
+      >
+        <svg
```

Comment:
Instead of inlining, move it icons folder.

I also think there is already a dropdown component created in the app and `dropdown-default` is not a good component name, not sure why we have two anyways